Sources/MeiliSearch/Client.swift

```diff
@@ -197,6 +197,23 @@ public struct MeiliSearch {
         self.documents.get(UID, identifier, completion)
     }
 
+    /**
+     Get the Document for the given `uid` and `identifier`.
+
+     - parameter UID:        The unique identifier for the Document's index to be found.
+     - parameter identifier: The document identifier for the Document to be found.
+     - parameter completion: The completion closure used to notify when the server
+     completes the query request, it returns a `Result` object that contains  `T` value.
+     If the request was sucessful or `Error` if a failure occured.
+     */
+    public func getDocument<T>(
+        UID: String,
+        identifier: Int,
+        _ completion: @escaping (Result<T, Swift.Error>) -> Void)
+        where T: Codable, T: Equatable {
+        self.documents.get(UID, String(identifier), completion)
```

Comment:
@ednofedulo Is this correct to transform the `Int` into a `String`?

Reply:
I don't see any aditional risk by doing this change, specially by the fact that this value is used by this:

```swift
let query: String = "/indexes/\(UID)/documents/\(identifier)"
```

Reply:
I don't see any risk involving this parsing, since the values of a Int variable is very limited compared to a String, which the function already accepts without any validation of value.

Reply:
Would this still work with an identifier that is a string? for example
```swift
 self.client.getDocument(
            UID: uid,
            identifier: "ab12cd"
        )
```

Reply:
the actual behaviour that accepts string is still as is